js/donations-visualiser.js

```diff
@@ -190,6 +190,9 @@ var data_request = d3.json("data/all_data.json")
         $("#loading-modal").modal('hide');
         processData(data);
 
+        // zoom out a ways initially
+        zoomTo(0.5).event(container);
```

Comment:
ok, perfect!

On 9 June 2016 at 18:46, Tim Osborn <notifications@github.com> wrote:

> In js/donations-visualiser.js
> <https://github.com/river-jade/donations-visualiser/pull/16#discussion_r66403019>
> :
>
> > @@ -190,6 +190,9 @@ var data_request = d3.json("data/all_data.json")
> >          $("#loading-modal").modal('hide');
> >          processData(data);
> >
> > +        // zoom out a ways initially
> > +        zoomTo(0.5).event(container);
>
> it does zoom twice.. in the next PR I attempt to blend them as much as
> possible, given that each render is unique..
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/river-jade/donations-visualiser/pull/16/files/b6bbbdd4134161b0660a37f2a4ece90507611432..2288b07a8c5204159959ebd3584a53eb326886f4#r66403019>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AArMkN3tKqVb3ZYnds5ijEBHDbGzYJsXks5qJ9L0gaJpZM4Ivg33>
> .
>